### PR TITLE
Correct title for collections view

### PR DESCRIPTION
**Pull request by mikeger**
# Issue

We supposed to show both title of conversation and title of the screen in navigation title item.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Collections/TwoLineTitleView.swift [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            subtitleLabel.trailing == selfView.trailing
+            subtitleLabel.bottom == selfView.bottom
+        }
+
+        CASStyler.default().styleItem(self)
+
+        setNeedsLayout()
+        layoutIfNeeded()
+        
+        let size = self.systemLayoutSizeFitting(CGSize(width: 320, height: 44))
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Collections/TwoLineTitleView.swift]: Why is this necessary

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Collections/TwoLineTitleView.swift]: I think the navigation bar is not too good with autolayout, if I would not do this the title view stays with zero frame

Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Collections/TwoLineTitleView.swift]: :/
